src/lib/components/Access/utils.js

```diff
@@ -104,8 +204,7 @@ export function embargoSection(embargo) {
     opacity: "0.5",
     cursor: "default !important"
   }
-  const disabled = embargo.is(Embargo.DISABLED) || embargo.is(Embargo.LIFTED);
-  const embargoStyle = disabled ? greyedStyle : {};
+  const embargoStyle = embargo.is(EmbargoState.DISABLED) ? greyedStyle : {};
```

Comment:
about the style: I think if we start putting also inline styles we will mess more the UX general styling and we will end up not knowing from where the css rules are coming from....I guess also for consistency we can define for lists are own "greyStyle" i.e disabled view I guess and use it widely...

Reply:
Would it be better if I move this style to invenio-app-rdm deposit.less and here I will set the class corresponding to it?

Reply:
I would say that this can be part of the `list.overrides` of the `rdm` theme as it is indeed a generic feature for the list components....On the other hand, it might be used only in this list so putting it in the `deposit.less` might also make sense...